drivers/adxl372/adxl372.h

```diff
@@ -204,6 +204,11 @@
 #define ADXL372_INT2_MAP_LOW_MSK		BIT(7)
 #define ADXL372_INT2_MAP_LOW_MODE(x)		(((x) & 0x1) << 7)
 
+#define ADXL372_SELF_TEST_ST_DONE_MSK		BIT(1)
+#define ADXL372_SELF_TEST_ST_DONE_MODE(x)	(((x) & 0x1) << 1)
+#define ADXL372_SELF_TEST_USER_ST_MSK		BIT(2)
+#define ADXL372_SELF_TEST_USER_ST_MODE(x)	(((x) & 0x1) << 2)
+
```

Comment:
I am not sure if these commands are of any use, since ST_DONE and USER_ST bits are read only and these masks are used together with adxl372_spi_write_mask() function.
However, we can add a command for initializing the self test, something like this:
```
#define ADXL372_SELF_TEST_ST_MSK	BIT(0)
#define ADXL372_SELF_TEST_ST_MODE(x)	(((x) & 0x1) << 0)
```

Reply:
If we want to check if ST_DONE and USER_ST bits are set, we can do it something like this: 
```
#define ADXL372_SELF_TEST_ST_DONE(x)		(((x) >> 1) & 0x1)
#define ADXL372_SELF_TEST_USER_ST(x)		(((x) >> 2) & 0x1)
```